src/laser/MultiLaser.cpp

```diff
@@ -1137,10 +822,35 @@ MultiLaser::InitLaserSlice (const int islice, const int comp)
     for ( amrex::MFIter mfi(m_slices, DfltMfiTlng); mfi.isValid(); ++mfi ){
         const amrex::Box& bx = mfi.tilebox();
         amrex::Array4<amrex::Real> const & arr = m_slices.array(mfi);
-        // Initialize a Gaussian laser envelope on slice islice
-        //check point
+        // Initialize a laser envelope on slice islice
         for (int ilaser=0; ilaser < m_nlasers; ilaser++) {
```

Comment:
If we want to have spacing consistency between `<` and `=` operator on this line.
```suggestion
        for (int ilaser = 0; ilaser < m_nlasers; ilaser++) {
```